src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbBase.java

```diff
@@ -329,7 +329,7 @@ public ListTablesResult listTables() {
 
     @Override
     public ListTablesResult listTables(String exclusiveStartTableName) {
-        throw new UnsupportedOperationException();
+        return getAmazonDynamoDb().listTables(exclusiveStartTableName);
```

Comment:
Shouldn't this only return tables visible to the current context? I.e., if you are calling with context "tenant1", only "tenant1" tables. And if you are calling with no context (your use case) only tables for which `isMtTable` returns true?